NEWS.md

```diff
@@ -6,6 +6,8 @@
 `AVALCATx` & `AVALCAxN`. (#2480)
 - New function `derive_vars_crit_flag()` for deriving criterion flag variables
 (`CRITy`, `CRITyFL`, `CRITyFLN`). (#2468)
+- New function `transform_scale()`  to transform values from a source range to a
```

Comment:
A new type of function to add to our list!?! https://pharmaverse.github.io/admiraldev/dev/articles/programming_strategy.html#function-names

Reply:
No, it's a computation. I.e., it is covered by
> `compute_` / `calculate_` / … | Functions that take vectors as input and return a vector

Reply:
should we add `transform_` so developers know they can use that as a verb?

Reply:
Then we would need to add `convert_` and `impute_` as well.

Reply:
I would be happy with that!

Reply:
OK, I have created an issue in admiraldev (https://github.com/pharmaverse/admiraldev/issues/472).